src/Symfony/Component/Serializer/Normalizer/PropertyNormalizer.php

```diff
@@ -150,4 +152,33 @@ protected function setAttributeValue($object, $attribute, $value, $format = null
 
         $reflectionProperty->setValue($object, $value);
     }
+
+    /**
+     * @param string|object $classOrObject
+     * @param string        $attribute
+     *
+     * @return \ReflectionProperty
+     *
+     * @throws \ReflectionException
+     */
+    private function getReflectionProperty($classOrObject, $attribute)
```

Comment:
What about:

```
    private function getReflectionProperty($classOrObject, $attribute)
    {
        $class = is_string($classOrObject) ? $classOrObject : get_class($classOrObject);
        while (true) {
            try {
                return isset($reflectionClass) ? $reflectionClass->getProperty($attribute) : new \ReflectionProperty($class, $attribute);
            } catch (\ReflectionException $e) {
                $reflectionClass = new \ReflectionClass($class);
                if (!$reflectionClass->getParentClass()) {
                    throw $e;
                }
            }
        }
    }
```

Reply:
This runs into an endless loop.

Reply:
Would work with this:

    ... 
    $reflectionClass = new \ReflectionClass($class);
    $reflectionClass = $reflectionClass->getParentClass();
    if ($reflectionClass === false) {
        throw $e;
    }
    ...

Reply:
I've updated my snippet, it should work now

Reply:
i updated the PR with the snippet included